packages/imperative/src/doc/IImperativeAutoInitCommandConfig.ts

```diff
@@ -0,0 +1,19 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { ICommandProfileAutoInitConfig } from "../../../cmd/src/doc/profiles/definition/ICommandProfileAutoInitConfig";
+
+export interface IImperativeAutoInitCommandConfig {
+    /**
+     * Command definition for "config auto-init" command
+     */
+    autoInitConfig?: ICommandProfileAutoInitConfig;
```

Comment:
Do we expect we may add more properties to this interface in the future?
If not, would it make sense to flatten the object structure to remove an unnecessary level?

Nested (current): `configAutoInitCommandConfig: {
    autoInitConfig: {...}
}`
Flattened (alternative): `configAutoInitCommandConfig: {...}`

Sorry this is probably a dumb question 😢

Reply:
I don't think we are planning on adding anything else in the near future.